transform/codon/codon.go

```diff
@@ -70,15 +71,15 @@ type AminoAcid struct {
 	Codons []Codon `json:"codons"`
 }
 
-// CodonTable holds information for a codon table.
-type CodonTable struct {
+// Table holds information for a codon table.
+type Table struct {
```

Comment:
`Table` seems less descriptive and grep'able than `CodonTable`.

Reply:
That's true but when used in this new subpackage configuration it's written as `codon.Table` in packages that depend on it so it'd likely be just as grepable. Does feel super weird in package though. I've got similar feelings about `random.ProteinSequence`.

Reply:
Yea probably fine now that we have subpackages.

Reply:
ah, if it's always written out as `codon.Table` that is pretty nice.